sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1252,7 +1253,38 @@ def moment_load_vector(self):
         """
         return self._moment_load_vector
 
-    def apply_load(self, value, dir="y", order=0):
+    @property
+    def boundary_conditions(self):
+        """
+        Returns a dictionary of boundary conditions applied on the beam.
+        The dictionary has two kewwords namely slope and deflection.
+        The value of each keyword is a list of tuple, where each tuple
+        contains loaction and value of a boundary condition in the format
+        (location, value). Further each value is a list corresponding to
+        slope or deflection(s) values along three axes at that location.
+
+        Examples
+        ========
+        There is a beam of length 4 meters. The slope at 0 should be 4 along
+        the x-axis and 0 along others. At the other end of beam, deflection
+        along all the three axes should be zero.
+
+        >>> from sympy.physics.continuum_mechanics.beam import Beam3D
+        >>> from sympy import symbols
+        >>> l, E, G, I, A, x = symbols('l, E, G, I, A, x')
+        >>> b = Beam3D(30, E, G, I, A, x)
```

Comment:
Doesn't a 3d beam have to have two second moments of area and a product moment of area?

Reply:
What about the second polar moment of area? Is this calculated from I somehow?

Reply:
Yeah, 3d beam has to have two second moments. It's just if `Ix = Iy`, we can pass only one value (Like `I` here) and both moments would get equated to it. And if they are unequal, then pass a `list` of the form `[Ix, Iy]` (Mentiond [here](https://github.com/sympy/sympy/blob/ff402664c8753c886c97335aa84635d1de3fe97e/sympy/physics/continuum_mechanics/beam.py#L1522) in the docstring).

Reply:
There is no special method for polar moment as of now. I think it would be just equal to `Ix + Iy`?